.github/meta.yaml

```diff
@@ -37,7 +37,7 @@ outputs:
         - requirements-parser >=0.2.0
         - shap >=0.40.0
         - texttable >=1.6.2
-        - woodwork <=0.19.0
+        - woodwork >=0.19.0,!=0.20.0
```

Comment:
Should be `!=0.21.0`, right? Since that's the version that introduces the breaking changes?

Reply:
Ah good call, it'll actually be both `0.20.0` and `0.21.0` that get blocked